GPUSimulators/EE2D_KP07_dimsplit.py

```diff
@@ -105,8 +105,8 @@ def __init__(self,
         self.cfl_data = gpuarray.GPUArray(self.grid_size, dtype=np.float32)
         dt_x = np.min(self.dx / (np.abs(rho_u/rho) + np.sqrt(gamma*rho)))
         dt_y = np.min(self.dy / (np.abs(rho_v/rho) + np.sqrt(gamma*rho)))
-        dt = min(dt_x, dt_y)
-        self.cfl_data.fill(dt, stream=self.stream)
+        self.dt = min(dt_x, dt_y)
```

Comment:
Are you sure this is correct? i don't think that dt should be a member variable here.

Reply:
In that case, the EulerTesting notebook must be changed, otherwise we get the following error when running the notebook:
"""
AttributeError                            Traceback (most recent call last)
<ipython-input-9-08c07a22a85e> in <module>
     10 arguments = InitialConditions.genShockBubble(nx, ny, gamma)
     11 arguments['context'] = my_context
---> 12 outfile = runSimulation(outfile, t_end, arguments)

<ipython-input-6-bc695c043f5b> in runSimulation(outfile, t_end, sim_args)
     40 
     41         progress_printer = Common.ProgressPrinter(n_save, print_every=10)
---> 42         print("Simulating to t={:f}. Estimated {:d} timesteps (dt={:f})".format(t_end, int(t_end / sim.dt), sim.dt))
     43         for i in range(n_save+1):
     44             #Sanity check simulator

AttributeError: 'EE2D_KP07_dimsplit' object has no attribute 'dt'
"""

Should we change the notebook or the simulator class?